### PR TITLE
fix / hide secret to .env

### LINKED ISSUE #3: Move all secrets to the .env file
Now the chat IDs are located right in the repository code. You need to move them to a file with secrets

### PR DESCRIPTION
**Pull request by Ouozen**
close #3 
this commit solves the problem of secrets lying in the public domain

### CODE REVIEW

File: readme.md
Comment (Dimildizio) [inline readme.md:1]: mb a good idea would be to create an empty template file with the same fields as .env but empty values?

File: main.py [hunk 1]
```diff
@@ -45,19 +52,19 @@ async def files(message: types.Message):
...
-        await bot.send_message(-1002027282215, message.from_user.username, message_thread_id=45)
-        await bot.send_document(-1002027282215, document=message.document.file_id, message_thread_id=45)
+        await bot.send_message(ARCHIVE, message.from_user.username, message_thread_id=FILES)
+        await bot.send_document(ARCHIVE, document=message.document.file_id, message_thread_id=FILES)
 
 
-# пересылает мемы из указанных групп
-@dp.message((F.chat.id == -1001675679569) & (F.forward_origin.chat.id.in_({-1001595506698, -1001081170974, -1001009232144, -1001399874898})))
+# forwards memes from the specified groups
+@dp.message((F.chat.id == ORIGIN) & (F.forward_origin.chat.id.in_({-1001595506698, -1001081170974, -1001009232144, -1001399874898})))
```
Comment (Dimildizio) [inline main.py:60]: memes list go in a separate config file?

Comment (Ouozen) [reply main.py:60]: I don't see much point, these are public groups. In any case, this rule will be changed soon. We plan to build normal rules on the data that the bot will collect.

File: readme.md
Comment (Ouozen) [reply readme.md:1]: There aren't many fields yet, so I didn't do it.

**Pull request merged**